src/orchard/resource.clj

```diff
@@ -0,0 +1,27 @@
+(ns orchard.resource
+  "Resolve JVM resource-related information."
+  {:added "0.5.0"}
+  (:require
+   [orchard.misc :as u]
+   [orchard.java.resource :as jresource]))
+
+(defn resource
+  "Return the resource path for the given input map.
+
+  It requires the :name key."
+  {:added "0.5.0"}
+  [{:keys [name]}]
+  {:resource-path
+   (when-let [resource (jresource/resource-full-path name)]
+     (.getPath resource))})
+
+(defn resources-list
+  "Return a list of dictionaries containing file and relpath: file is the
+  absolute path to the resource, relpath is the path of the resource
+  relative to the classpath."
+  {:added "0.5.0"}
+  [_]
+  {:resources-list
+   (u/transform-value
```

Comment:
This is needed only in the context of nREPL responses. Not sure if it makes sense to transform any values for nREPL in Orchard.

Reply:
Yeah well I never understood this transform really. It is kind of called in a lot of places and if that's the case (presentation only) probably it should not live in orchard but in cider-nrepl?

I will check what it does

Reply:
I don't remember what exactly it did, but it was just converting values to types that bencode could represent. I think I copied it accidentally to Orchard and then forgot to move it back.

Reply:
Oh ok good to know, good target for another PR